demo/get-multicluster-envoy-configs.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This script automates the collection of Envoy config during Multicluster demo/development.
+# The configs are collected from Envoys in the ALPHA kube context (cluster).
+
+kubectx alpha > /dev/null
```

Comment:
lets add a not the the script stating that one needs `kubectx` to run this script

Reply:
That's a good point.  I should use `kubectl` instead perhaps.

Reply:
Switched to `kubectl config use-context alpha`